nsqd/http.go

```diff
@@ -494,7 +494,24 @@ func (s *httpServer) doStats(w http.ResponseWriter, req *http.Request, ps httpro
 	channelName, _ := reqParams.Get("channel")
 	jsonFormat := formatString == "json"
 
-	stats := s.ctx.nsqd.GetStats()
+	var (
+		state TopicStats
+		stats []TopicStats
```

Comment:
the `state` variable is awkwardly named, how about we call these `stats TopicStats` and `allStats []TopicStats`?